x/avs/types/types.go

```diff
@@ -1 +1,19 @@
 package types
+
+func ContainsString(slice []string, target string) bool {
```

Comment:
No need to re-implement it, just use `import slices` and `slices.Contains()`

Reply:
fixed